quedex_api/user_stream.py

```diff
@@ -415,6 +443,8 @@ def check_modify_order(modify_order):
   if 'new_limit_price' not in modify_order and 'new_quantity' not in modify_order:
     raise ValueError('modify_order should have new_limit_price or new_quantity')
 
+def check_cancel_all_orders(cancel_all_orders):
+  return True
```

Comment:
Other `check_*` methods don't return anything, seams misleading and redundant...?

Reply:
OK, I'll change it to 'pass'